pages/workflow/review.mdx

```diff
@@ -11,7 +11,7 @@ import { Callout } from 'nextra/components';
 
 _Role: Product, Data, Engineering. See more below_
 
-Once you have drafted changes to your tracking plan we recommend asking your teammates to review your suggested changes.
+Once you have drafted changes to your tracking plan we recommend asking your teammates to review your suggested changes. To do this, click "View changes" in the branch header to navigate to the review screen.
```

Comment:
Add this image here: https://www.figma.com/design/clTN5ju15PBz1yjxJJFAai/Workspace-domains?node-id=1174-6084&t=4QDqvOW8y0NUN1jp-1

Reply:
Are you asking for help to add the image @klaraavo ?

Reply:
Yeah, that was before I got onboarded into managing docs natively. But some quick support would be lovely